source/fundamentals/typescript.txt

```diff
@@ -104,22 +106,63 @@ genus and color of dogs and cats:
      classification: { genus: "Canis" | "Felis"; color: string };
    }
 
-The following code snippet correctly raises a type error when specifying
-the genus of an unsupported animal in a query:
+The following code snippet correctly raises a type error when setting
+the ``classification.color`` to a ``number`` type in an update
+operation:
 
 .. code-block:: typescript
 
-   database
-     .collection<ClassificationPet>("<your collection>")
-     .find({ "classification.genus": "Sylvilagus" });
+   await collection.updateOne({ name: "Garfield" }, { $set: { "classification.color": 101 } });
 
 The type error raised by the preceding code snippet is as follows:
 
 .. code-block:: none
 
-   No overload matches this call.
-   ...
-   Type '"Sylvilagus"' is not assignable to type 'Condition<"Canis" | "Felis">'.
+   Type 'number' is not assignable to type 'string'.
+   
+Dot Notation in Find Operations
+```````````````````````````````
+
+Starting in version 5.0, by default, the {+driver-short+} does not provide type
+safety for find operations that search on fields expressed in dot
+notation. When you construct a filter to pass to a find method, the
+driver will not raise a type error even if you specify an incorrectly
+typed value for a field expressed in dot notation.
+
+.. note::
+
+   The following methods comprise find operations:
+
+   - ``find()``
+   - ``findOne()``
+   - ``findOneAndDelete()``
+   - ``findOneAndReplace()``
+   - ``findOneAndUpdate()``
```

Comment:
Suggestion:

For consistency, we should standardize "find operations" to match this list for all drivers. For example, the golang docs only mention find and findOne:
https://github.com/mongodb/docs-golang/blob/master/source/fundamentals/crud/read-operations/retrieve.txt#L59-L60

Reply:
changed the language for this list, but agreed- this should be standardized. Making a ticket for myself.